ublox/src/ubx_packets/packets.rs

```diff
@@ -6,9 +6,11 @@ use crate::error::{MemWriterError, ParserError};
 use bitflags::bitflags;
 use chrono::prelude::*;
 use core::fmt;
+use core::fmt::Formatter;
 use num_traits::cast::{FromPrimitive, ToPrimitive};
 use num_traits::float::FloatCore;
 use num_traits::real::Real;
+use std::convert::TryInto;
```

Comment:
You should be importing things from `core` instead of `std`, assuming we ever want to merge these changes upstream. this crate has 2 features/3 modes

- `std` (the default)
- `alloc` - for `no_std` but you have an allocator  
- no feature - `no_std` and no allocator 

All three support stuff inside `core`. `alloc` is supported by `alloc` and `std`. only `std` can use `std`. All of the messages need to support running with no features (hence the iterators instead of vecs for some of the fields). The features seem to only enable extra things in the parser

Reply:
We can fix all that kind of stuff tho after so no need to do it now